database/factories/ShowFactory.php

```diff
@@ -4,6 +4,7 @@
 
 use App\Models\Show;
 use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
```

Comment:
I do not think this is being used right? If that is the case, this can be deleted hehe.

Otherwise, it is usually recommended to organise the imports according to their length.